tests/fileTypeTests.py

```diff
@@ -123,6 +124,13 @@ def test_award_fin_mixed(self):
         jobId = self.jobIdDict["awardFinMixed"]
         self.passed = self.run_test(
             jobId, 200, "finished", 17206, 5, "complete", 87, True)
+        # Test that whitespace is converted to null
+        rowThree = self.interfaces.validationDb.session.query(AwardFinancial).filter(AwardFinancial.row_number == 3).filter(AwardFinancial.submission_id == self.interfaces.jobDb.getSubmissionId(jobId)).first()
```

Comment:
Now checking against parentawardid instead of row number, so inserting or deleting rows won't affect these tests.